### DISCUSSION TITLE
触摸板支持

### DISCUSSION THREAD

**Discussion by Hugeria** [shuax/MouseInc.Settings#258] [Ideas]
神一样的软件，强烈希望追加触摸板支持功能，对2-5个手指的手势做出支持，救救需要在外办公的笔记本电脑吧！

具体来说有以下常用手势：

两指，三指，四指
- 上划
- 下划
- 左划
- 右划
- 单击
- 双击

还有比如说在使用触摸板的前提下可以支持两个手指直接画手势就实在是太方便了，实在没办法按住右键

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/162605417-42ceec44-0ca7-4459-9584-a561238c0633.png)
我感觉系统带的够用了。

**Reply by Hugeria** [reply]
当只能用触摸板的时候，系统自带的就非常不够了

每个都设置自定义按键然后用autohotkey来写可以勉强达到可以用的程度，但是可以做的事情比起鼠标手势实在是少太多了(T_T)

**Comment by shuax** [comment]
我还是觉得鼠标好用
